engine/src/main/java/org/terasology/engine/subsystem/config/BindsManager.java

```diff
@@ -0,0 +1,49 @@
+
+
+package org.terasology.engine.subsystem.config;
+
+import org.terasology.config.BindsConfig;
+import org.terasology.context.Context;
+import org.terasology.engine.SimpleUri;
+import org.terasology.engine.module.ModuleManager;
+import org.terasology.input.BindableButton;
+import org.terasology.input.ControllerInput;
+import org.terasology.input.Input;
+import org.terasology.input.InputSystem;
+import org.terasology.input.MouseInput;
+import org.terasology.input.internal.AbstractBindableAxis;
+import org.terasology.input.internal.BindableRealAxis;
+
+import java.util.List;
+import java.util.Map;
+
+public interface BindsManager {
+
+    BindsConfig getBindsConfig();
+    
+    BindsConfig createDefault(Context context);
+    
+    void updateForChangedMods(Context context);
```

Comment:
Just for consistency's sake should this be `Modules` rather than `Mods` ? Historically we've tried to make (container) modules distinct from (gameplay) mods. Probably a somewhat doomed battle, I admit.